assets/new-user-experience/onboarding/onboarding-app.js

```diff
@@ -0,0 +1,215 @@
+/**
+ * External dependencies.
+ */
+import { useEffect, useState } from '@wordpress/element';
+
+/**
+ * Internal dependencies.
+ */
+import { useSquareSettings } from '../settings/hooks';
+import { usePaymentGatewaySettings, useSteps } from './hooks';
+import {
+	CashAppSetup,
+	ConnectSetup,
+	CreditCardSetup,
+	DigitalWalletsSetup,
+	GiftCardSetup,
+	PaymentMethods,
+	BusinessLocation,
+	PaymentComplete,
+} from './steps';
+import { ConfigureSync, AdvancedSettings, SandboxSettings } from '../modules';
+
+import { OnboardingHeader, PaymentGatewaySettingsSaveButton, SquareSettingsSaveButton, Loader } from '../components';
+
+export const OnboardingApp = () => {
+	const [ settingsLoaded, setSettingsLoaded ] = useState( false );
+
+	const {
+		paymentGatewaySettingsLoaded,
+		cashAppGatewaySettingsLoaded,
+		giftCardsGatewaySettingsLoaded,
+		savePaymentGatewaySettings,
+		saveGiftCardsSettings,
+		saveCashAppSettings,
+	} = usePaymentGatewaySettings( true );
+
+	const {
+		stepData,
+		setStep,
+		setBackStep,
+	} = useSteps( true );
+
+	const { step, backStep } = stepData;
+
+	const useSquareSettingsData = useSquareSettings( true );
+
+	const {
+		settings,
+		squareSettingsLoaded
+	} = useSquareSettingsData;
+
+	// Set info in local storage.
+	useEffect(() => {
```

Comment:
Seeing a mix of styling in this file, some places don't have spaces (like here) but others do. We should standardize that to match what we're doing elsewhere (looks like adding spaces is the desired outcome)